src/components/search/Locator.tsx

```diff
@@ -100,10 +101,10 @@ const Locator = (props: LocatorProps) => {
             <Map
               provider={GoogleMaps}
               providerOptions={{ styles: mapStyles }}
-              clientKey="gme-yextinc"
```

Comment:
I'm fairly sure they don't issue new clientKeys anymore right? No need to support apikey AND clientKey?

Reply:
Yeah we can remove that option, but that change would be in the component repo library we import the map component from.